src/operator/tensor/broadcast_reduce_op.h

```diff
@@ -1021,8 +1023,38 @@ void L2NormGradCompute(const nnvm::NodeAttrs& attrs,
   } else {
     small = ReduceAxesShapeImpl(outputs[0].shape_, param.axis, true, false);
   }
-  ReduceAxesBackwardUseInOutImpl<xpu, mshadow_op::div, false>(ctx, small, inputs,
+  if (param.ord == 1) {
+    using namespace mshadow;
+    using namespace mshadow::expr;
```

Comment:
These to `using namespace` line are redundant.

Reply:
shall remove it.